src/main/resources/lang/lang_de_DE.properties

```diff
@@ -1066,6 +1066,9 @@ CommandFactionAddMemberUnknownNewPlayerFaction=Unbekanntes neues Mitglied
 CommandFactionAddMemberSuccess=Dieser Spieler ist jetzt Mitglied der angegebenen Fraktion.
 CommandFactionHelpAddMember=/faction addmember <player> <faction> - F�gt zwangsweise ein Mitglied zu einer Fraktion hinzu.
 CommandFactionRoleSetPermissionCannotModifyRolePermissionToModifyOwnRolePermission=Du kannst die Berechtigung deiner Rolle nicht �ndern, um die Berechtigungen deiner eigenen Rolle zu �ndern.
+DeprecationWarningAutoclaim=Warnung: Die automatische Anspruchsfunktion wird in einer zuk�nftigen Version entfernt. Verwenden Sie stattdessen /f claim auto.
```

Comment:
Maybe 'Achtung' over 'Warnung', also use impolite form (everything else uses 'du')
Pass this by our translators maybe?

Reply:
I had ChatGPT generate these when usually I use Google Translate. That probably explains the discrepancy.

Reply:
I've posted this in the translator chat.

Google Translate suggested the following:

```
DeprecationWarningAutoclaim=Befehl veraltet. Verwenden Sie stattdessen /fclaim auto.
DeprecationWarningFillclaim=Befehl veraltet. Verwenden Sie stattdessen /f Claim Fill.
DeprecationWarningCheckclaim=Befehl veraltet. Verwenden Sie stattdessen die Anspruchsprüfung /f.
```

Reply:
What about:
DeprecationWarningAutoclaim=Befehl veraltet. Verwendest du stattdessen /f claim auto. DeprecationWarningFillclaim=Befehl veraltet. Verwendest du stattdessen /f claim fill.
DeprecationWarningCheckclaim=Befehl veraltet. Verwendest du stattdessen /f claim check.

Reply:
Sounds good to me!

Reply:
Your suggested translations have been implemented :)